tests/gluon_autolog/test_gluon_autolog.py

```diff
@@ -0,0 +1,73 @@
+import random
+import warnings
+
+import mxnet.ndarray as nd
```

Comment:
I think we'll need to add mxnet to test requirements as well. CI is currently failing.

Reply:
I'll do that.